src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
 // write your code here
+const rows = document.querySelectorAll('table tr');
+
+for (const i of rows) {
+  const lastChild = i.querySelector(':last-child');
+  const secondChild = i.querySelector(':nth-child(2)');
+  const copySecondchild = document.createElement(secondChild.tagName);
+
+  copySecondchild.innerText = secondChild.innerText;
```

Comment:
Similar to the previous comment, ensure that `secondChild` is not null before accessing its `innerText` to avoid runtime errors.